proto/grpc/device_testing.proto

```diff
@@ -0,0 +1,22 @@
+/*
+ * gRPC service to receive device testing results
+ */
+
+syntax = "proto3";
+
+import "forch/proto/shared_constants.proto";
+
+/*
+ * gRPC service to receive testing results
+ */
+service DeviceTesting {
+    rpc ReportTestingResult (DeviceTestingResult) returns (Empty) {}
+}
+
+/*
+ * Device testing result
+ */
+message DeviceTestingResult {
+    string mac = 1;
+    bool passed = 2;
```

Comment:
make this an enum not a bool. Initial entries can be { unknown, testing, passed } -- it will just give us more room in the future to expand to include other states -- e.g. when an anomaly is detected or the device is explicitly cleared.

Reply:
Added TestingState enumus.